src/explore-education-statistics-admin/src/pages/users/ManageUserPage.tsx

```diff
@@ -0,0 +1,174 @@
+import Link from '@admin/components/Link';
+import Button from '@common/components/Button';
+import ButtonText from '@common/components/ButtonText';
+import { errorCodeToFieldError } from '@common/components/form/util/serverValidationHandler';
+import {
+  FormFieldset,
+  FormFieldSelect,
+  FormFieldCheckboxGroup,
+  FormFieldTextInput,
+  FormGroup,
+  Formik,
+} from '@common/components/form';
+import Form from '@common/components/form/Form';
+import Page from '@admin/components/Page';
+import useFormSubmit from '@admin/hooks/useFormSubmit';
+import userService from '@admin/services/users/service';
+import { UserStatus } from '@admin/services/users/types';
+import useAsyncRetry from '@common/hooks/useAsyncRetry';
+import Yup from '@common/validation/yup';
+
+import React, { useEffect, useState } from 'react';
+import { Route, RouteComponentProps, Switch } from 'react-router';
+import { IdLabelPair } from 'src/services/common/types';
+
+const errorCodeMappings = [
+  errorCodeToFieldError(
+    'USER_ALREADY_EXISTS',
+    'userEmail',
+    'User already exists',
+  ),
+];
+
+interface Model {
+  user: UserStatus;
+}
+
+interface FormValues {
+  userEmail: string;
+}
+
+const ManageUserPage = ({
```

Comment:
This page is WIP. Disabled link to access it and currently doesn't perform any actions.